test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Infrastructure/PageDirectiveFeatureTest.cs

```diff
@@ -48,7 +48,7 @@ public void TryGetPageDirective_JunkBeforeDirective()
             Assert.Null(template);
         }
 
-        [Fact]
+        [Fact(Skip = "This fails")]
```

Comment:
Kept it in case we super care about supporting this. I'm not sure you'd want to support string tokens with quotes in them. And if you truly wanted a quoted route, you could use `AddPageRoute`.